docker/server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
```

Comment:
done, `alpine` is feasible if we don't use `apt-get`, which we don't really need.